### PR TITLE
Start servers fix

### LINKED ISSUE #125: PR #117 broke polyserve on the CLI
Running `polyserve` with the CLI is broken since #117 has been merged.
I have bisected from the previous release to master and the merge commit of #117 has caused that running `./bin/polyserve` directly terminates after logging `no bower.json detected, using name "serve"` to the console. Polyserve is not running and can not process any requests.

### PR DESCRIPTION
**Pull request by justinfagnani**
Fixes #125

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -67,7 +67,8 @@ export async function run(): Promise<void> {
   const serverInfos = await startServers(options);
 
   if (serverInfos.kind === 'mainline') {
-    const urls = getServerUrls(options, serverInfos[0]!.server);
+    const mainlineServer = serverInfos;
```
Comment (justinfagnani) [inline src/cli.ts]: This is the actual fix. The rests is testing.

File: src/test/cli_test.ts [hunk 2]
```diff
@@ -40,8 +41,21 @@ suite('cli', () => {
...
 
+    const closeServer = (server: http.Server) =>
+        new Promise((resolve, reject) => server.close((e: Error) => {
+          e ? reject(e) : resolve();
+        }));
+
     try {
-      await cliRun();
+      const serverInfos = await cliRun();
+      if (serverInfos) {
```
Comment (justinfagnani) [inline src/test/cli_test.ts]: Now that there's a test that tries to successfully launch some servers, we need to shut them down and wait or else we get port-in-use errors.

File: src/cli.ts (hunk 1 above)
Comment (rictic) [inline src/cli.ts]: Ouch. I guess we get that from not having strict indexing types. You don't need to do this assignment, though it does help legibility.

**Comment by TimvdLippe** [comment]
Described a different issue regarding `filePath` resolution with `send()` in #125 too, but at least this PR fixes invoking the CLI.

**Pull request merged**